deno_webgpu/device.rs

```diff
@@ -419,16 +422,20 @@ impl GPUDevice {
         let (id, err) = self.instance.device_create_shader_module(
             self.id,
             &wgpu_descriptor,
-            wgpu_core::pipeline::ShaderModuleSource::Wgsl(Cow::Owned(descriptor.code)),
+            wgpu_core::pipeline::ShaderModuleSource::Wgsl(Cow::Borrowed(&descriptor.code)),
             None,
         );
 
+        let info1 = GPUCompilationInfo::new(scope, err.iter(), &descriptor.code);
+        let info2 = make_cppgc_object(scope, info1);
+        let compilation_info = v8::Global::new(scope, info2);
```

Comment:
suggestion(non-blocking): I think it's actually easier for a reader to see multiple `let compilation_info = …` statements in a row, rather than single-use one-off identifiers that eventually culminate in a `let compilation_info = …` anyway.